test_b2_command_line.py

```diff
@@ -680,6 +680,37 @@ def _sync_test_using_dir(b2_tool, bucket_name, dir_):
             ], file_version_summary(file_versions)
         )
 
+        # confirm symlink is skipped and reported as such
+        write_file(p('linktarget'), b'hello')
+        os.symlink('linktarget', p('alink'))
+
+        b2_tool.should_succeed(
+            ['sync', '--noProgress', '--excludeAllSymlinks', dir_path, b2_sync_point],
+            expected_pattern="skipped symlink .*/alink"
+        )
+        file_versions = b2_tool.list_file_versions(bucket_name)
+        should_equal(
+            [
+                '+ ' + prefix + 'c',
+                '+ ' + prefix + 'c',
+                '+ ' + prefix + 'c',
+                '+ ' + prefix + 'linktarget',
+            ], file_version_summary(file_versions)
```

Comment:
Optional style thing: Python allows a trailing comma in arg lists, and having one makes yapf format more nicely.  In this case, it would put `file_version_summary` on a new line.

Reply:
This differs from the style used in the prior calls to `should_equal`, is that OK?